cartoframes/io/managers/context_manager.py

```diff
@@ -290,7 +294,9 @@ def _truncate_table(self, table_name, schema, cartodbfy):
 
     def _truncate_and_drop_add_columns(self, table_name, schema, df_columns, table_columns, cartodbfy):
         log.debug('TRUNCATE AND DROP + ADD columns table "{}"'.format(table_name))
-        query = 'BEGIN; {truncate}; {drop_columns}; {add_columns}; {cartodbfy}; COMMIT;'.format(
+        regExists = self._check_regenerate_table_exists()
```

Comment:
As `regExists` is only used in the next line we could avoid creating the variable here and just calling `self._check_regenerate_table_exists()` in the next line